src/seedsigner/helpers/embit_utils.py

```diff
@@ -30,6 +30,8 @@ def get_standard_derivation_path(network: str = SettingsConstants.MAINNET, walle
         raise Exception("Unexpected network")
 
     if wallet_type == SettingsConstants.SINGLE_SIG:
+        if is_electrum:
```

Comment:
Wondering if this (and the multisig one below) should be nested under the native-segwit branch just below?

Reply:
Yes, this may be the best route considering there may be a way to enable nested segwit (currently possible, just need to figure out how) and taproot (not currently supported in electrum) for electrum seeds.

Will make this suggested change with some other mods.

Reply:
this change has been made